packages/admin/cms-admin/src/pages/pagesPage/PagesPage.tsx

```diff
@@ -80,6 +84,22 @@ export function PagesPage({
         stopPolling,
     });
 
+    if (error) {
+        const isPollingError = data !== undefined;
```

Comment:
another situation where this can happen is data can be loaded from apollo cache (possibly outdated)

Reply:
> would you differentiate between these two situations?

mainly this was regarding the isPollingError variable name. But when I think about it: silencing (i.e. not showing at all) polling errors is perfectly fine. Silencing the initial re-load-error when navigating to the page tree is a different thing.

> if yes, how?

No idea if that is even possible

Reply:
> Silencing the initial re-load-error when navigating to the page tree is a different thing.

Fixed in b2bc0cbc73268389702408b495e20e52f6cbdbae. Had to change the fetch policy to cache-and-network, otherwise the "stale" data was loaded from the cache without attempting to reload it. What do you think of this approach?